portfolio/src/main/webapp/script.js

```diff
@@ -87,3 +87,33 @@ async function getRandomQuoteUsingAsyncAwait() {
   document.getElementById('quote-container').innerText = quote;
 }
 
+/**
+ * Fetches quotess from the servers and adds them to the DOM.
+ */
+function getQuote() {
+  fetch('/data').then(response => response.json()).then((quotes) => {
+    // stats is an object, not a string, so we have to
+    // reference its fields to create HTML content
+
+    const statsListElement = document.getElementById('quote-container');
+    statsListElement.innerHTML = '';
+    statsListElement.appendChild(
+        createListElement('First Quote: ' + quotes.firstQuote));
+    statsListElement.appendChild(
+        createListElement('Second Quote: ' + quotes.secondQuote));
+    statsListElement.appendChild(
+        createListElement('Third Quote: ' + quotes.thirdQuote));
+    statsListElement.appendChild(
+        createListElement('Fourth Quote: ' + quotes.fourthQuote));
+    statsListElement.appendChild(
+        createListElement('Fifth Quote: ' + quotes.fifthQuote));
```

Comment:
I just took a look at this again, unlike the example in "ServerStats," quotes as an object does not have these fields (firstQuote, secondQuote, etc.), instead it is an array. So you should be able to refer to it as quotes[index].